Resources/config/dbal.xml

```diff
@@ -23,10 +23,8 @@
         <service id="Doctrine\DBAL\Driver\Connection" alias="database_connection" public="false" />
         <service id="Doctrine\DBAL\Connection" alias="database_connection" public="false" />
 
-        <service id="doctrine.dbal.logger.chain" class="%doctrine.dbal.logger.chain.class%" public="false" abstract="true">
-            <call method="addLogger">
-                <argument type="service" id="doctrine.dbal.logger" />
-            </call>
+        <service id="doctrine.dbal.logger.chain" class="%doctrine.dbal.logger.chain.class%" public="false" abstract="true" >
+            <!-- TODO: deprecate this service in 2.3.0 -->
```

Comment:
@stof introducing a new deprecation is minor, so I should do it in a subsequent PR to master I think.